drivers/volume/portworx/portworx.go

```diff
@@ -252,6 +260,17 @@ func (p *portworx) GetPodVolumes(pod *v1.Pod) ([]*storkvolume.Info, error) {
 	return volumes, nil
 }
 
+func (p *portworx) GetStatefulSetTemplates(ss *v1beta1.StatefulSet) (
```

Comment:
That assumes that the PVC has been bound. Here we need to know if any of the templates are using the driver since this is being used by the initializer and will be called before the PVCs are even bound.